inst/rmarkdown/templates/iop/resources/template.tex

```diff
@@ -0,0 +1,122 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Author template for Institute of Physics (IOP) Journal articles
+%% Adapted for rticles by Rob J Hyndman Rob.Hyndman@monash.edu. Dec 2021
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\documentclass[$if(fontsize)$$fontsize$$else$$if(twocolumn)$10pt$else$12pt$endif$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{iopart}
+
+%% BEGIN MY ADDITIONS %%
+\pdfminorversion=4
+\usepackage{hyperref, graphicx, booktabs}
+% Bibliography allowing natbib with Harvard
+\usepackage{har2nat}  % loads 'natbib' automatically
+\bibliographystyle{dcu}
```

Comment:
This enforce the Harvard style in the template right ? 

Guidelines seems to also allow for a Vancouver style. Can this be changed ? 

> \item For the numerical (Vancouver) reference style we recommend that authors use 
 \verb"unsrt.bst"; this does not quite follow the style of published articles in our
 journals but this is not a problem.  Alternatively \verb"iopart-num.bst" created by Mark A Caprio
 produces a reference style that closely matches that in published articles.  The file is available from
\verb"http://ctan.org/tex-archive/biblio/bibtex/contrib/iopart-num/" .

Shouldn't this be configurable (even if a default is set ?) to use `unsrt.bst` or `iopart-num.bst` ? 

Could be with a default 

```
\bibliographystyle{$if(biblio-style)$$biblio-style$$else$dcu$$endif$}
```

Or more generic with settings in the YAML header: 
```
\bibliographystyle{$biblio-style$}
```
and
```yaml
# use dcu for alphabetical (Harvard) style references, 
# use unsrt or iopart-num for the numerical (Vancouver) reference style
biblio-style: dcu 
```

Reply:
Done